src/cairoWriter/writers/functionCallWriter.ts

```diff
@@ -29,7 +29,9 @@ export class FunctionCallWriter extends CairoASTNodeWriter {
     const func = writer.write(node.vExpression);
     switch (node.kind) {
       case FunctionCallKind.FunctionCall: {
-        if (node.vExpression instanceof MemberAccess) {
+        if (['assert', 'require', 'revert'].includes(func)) {
+          return [`assert(${args}${node.vArguments.length === 1 ? ", 'Assertion error'" : ''})`];
```

Comment:
Imo it should be done on the AST level. Could we create assert function call in builtin handler instead?